ingestion-beam/src/test/java/com/mozilla/telemetry/SinkTest.java

```diff
@@ -59,7 +60,7 @@ public void mainCanWriteToStdout() {
     });
   }
 
-  @Test
+  @Test(expected = PipelineExecutionException.class)
```

Comment:
I originally intended for this test to declare an expected exception, but realized it never did throw what I intended. It wasn't interpreting `${some_attribute}` as a placeholder. Now it does and it throws as expected.